pandas/tests/tools/test_merge.py

```diff
@@ -1355,3 +1355,51 @@ def test_dtype_on_merged_different(self, change, how, left, right):
                            np.dtype('int64')],
                           index=['X', 'Y', 'Z'])
         assert_series_equal(result, expected)
+
+
+@pytest.fixture
+def df1():
```

Comment:
@jreback I get an error when naming them left and right, because these names are also used in other parts of the code. What should I do?